frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable no-unused-vars, no-undef */
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+import { K8sKind, k8sList, referenceForModel } from '../../module/k8s';
+import { ResourceLink, resourcePathFromModel } from '../utils/resource-link';
+
+const globalConfigListStateToProps = ({k8s}) => ({
+  configResources: k8s.getIn(['RESOURCES', 'configResources']),
+});
+
+const ItemRow = ({item}) => {
+  return <div className="row co-resource-list__item">
+    <div className="col-sm-5 col-xs-6">
+      <ResourceLink kind={referenceForModel(item.model)} name={item.metadata.name} />
+    </div>
+    <div className="col-sm-5 col-xs-6">
+      {item.kind}
+    </div>
+    <div className="col-sm-2 hidden-xs">
+      <Link to={`${resourcePathFromModel(item.model)}/${item.metadata.name}/yaml`} className="btn btn-default pull-right">Edit YAML</Link>
+    </div>
+  </div>;
+};
+
+class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps> {
```

Comment:
We should add a state definition, too.

```suggestion
class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps, GlobalConfigPageState> {
```